perun-engine/src/main/java/cz/metacentrum/perun/engine/scheduling/impl/GenWorkerImpl.java

```diff
@@ -44,7 +44,7 @@ public Task call() throws TaskExecutionException {
 		getTask().setGenStartTime(new Date(System.currentTimeMillis()));
 		Service service = getTask().getService();
 
-		log.info("[{}] Executing GEN worker for Task with Service ID: {} and Facility ID: {}.",
+		log.debug("[{}] Executing GEN worker for Task with Service ID: {} and Facility ID: {}.",
```

Comment:
Please keep this in info level. Right now engine consistently logs processing of each task, so you can grep logs by ``"[taskId]"`` and see what happened to it. And any state change is usually in info level, so we don't have to enable debug to see where it failed.